utils/build-presets.ini

```diff
@@ -169,6 +169,17 @@ skip-test-ios-host
 skip-test-tvos-host
 skip-test-watchos-host
 
+
+[preset: buildbot,tools=RA,stdlib=RDA,long_test]
+mixin-preset=
+    mixin_buildbot_tools_RA_stdlib_RDA
+
+test=0
+validation-test=0
+long-test=1
```

Comment:
Oh! I see you've addressed this concern in the argument parser directly, with `argparse_bool`. Thanks for that!! 💯